tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/quorum/ZookeeperPluggableQuorumPeerTest.java

```diff
@@ -48,15 +47,7 @@ public ZookeeperPluggableQuorumPeerTest() {
       // both roles as both wish to be primary but will revert to backup
       primary = new BrokerControl("primary-peer-a", JMX_PORT_PRIMARY, "zkReplicationPrimaryPeerA", PRIMARY_PORT_OFFSET);
       backup = new BrokerControl("primary-peer-b", JMX_PORT_BACKUP, "zkReplicationPrimaryPeerB", BACKUP_PORT_OFFSET);
-      brokers = new LinkedList(Arrays.asList(primary, backup));
-   }
-
-   @Test
-   @Override
-   public void testBackupFailoverAndPrimaryFailback() throws Exception {
-      // peers don't request fail back by default
-      // just wait for setup to avoid partial stop of zk via fast tear down with async setup
-      Wait.waitFor(this::ensembleHasLeader);
```

Comment:
Again, did something change on the Curator side, or is this now potentially doing the very 'partial stop of zk via fast tear down with async setup' this was specifically there to avoid?

Reply:
reverted with different check. In this PR quorum tests are enabled to show that its passed